codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Stream.java

```diff
@@ -44,6 +61,26 @@
     State state();
 
     /**
+     * Get the state as related to the {@link Http2LocalFlowController}.
+     */
+    FlowControlState localFlowState();
+
+    /**
+     * Set the state as related to the {@link Http2LocalFlowController}.
+     */
+    void localFlowState(FlowControlState state);
```

Comment:
@nmittler - Makes sense.  With the reduction in scope of this PR it is easier to reason about and easier for you to digest in the properties PR I hope....perhaps we should pull this PR in and you can absorb into properties PR?